README.md

```diff
@@ -1,24 +1,41 @@
-# README
+## Best Blog Ever
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+This is the best blog app ever app, it was build with love using RoR.
 
-Things you may want to cover:
+# Built With
 
-* Ruby version
+- Ruby
+- Ruby on Rails
+- Tailwind CSS
+- PostgreSQL
 
-* System dependencies
 
-* Configuration
+# Getting Started
 
-* Database creation
+1. Clone the project using git-bash or Githup Desktop.
```

Comment:
- [ ] Let's update our readme file and include commands required for cloning the repo and all other commands required for running the project on the user's local machine.

Reply:
Hi, @elisha2kyakpo1 ! Thanks for the comments. I think your suggestion is an optional change, since my project fulfills all actual requirements. Thank you!

Reply:
Hey @danalvrz, thanks for reaching out, writing a professional readme file is one of the general requirements of any project at Microverse. Every time you have a project remember to have a readme file with all the commands necessary for cloning the repo and for running the project on the user's local machine, and not instructions.

Reply:
@elisha2kyakpo1 would you be kind and share with me your documentation for a "professional readme"? Because I considered mine was. Thanks!